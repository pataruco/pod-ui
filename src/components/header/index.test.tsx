import { shallow } from 'enzyme';
import React from 'react';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import Header, { HeaderComponent } from './';

describe('HeaderComponent', () => {
  it('renders', () => {
    const defaultProps = {
      fetchDataFn: () => ({}),
    };
    expect(shallow(<HeaderComponent {...defaultProps} />)).toMatchSnapshot();
  });
});

describe('Connected header', () => {
  const store = configureStore([])({});
  it('renders', () => {
    expect(
      shallow(
        <Provider store={store}>
          <Header />
        </Provider>,
      ),
    ).toMatchSnapshot();
  });
});
