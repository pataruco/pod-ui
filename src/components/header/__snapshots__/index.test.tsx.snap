// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Connected header renders 1`] = `<Connect(HeaderComponent) />`;

exports[`HeaderComponent renders 1`] = `
<header
  className="header"
>
  <h1
    className="title"
  >
    Peter of the day
  </h1>
</header>
`;
